api/controllers/interventions.js

```diff
@@ -0,0 +1,23 @@
+const Intervention = require('../models/intervention');
+
+function getIntervention(req, res) {
+  const id = req.swagger.params.id.value;
+
+  return new Intervention({ id: id }).fetch({ withRelated: Intervention.relatedModels })
```

Comment:
`Intervention.relatedModels` is undefined, so you can change it to just `.fetch()` or `.fetch({})` (if you have problems). The same for all other models.